content/change_notes/2.3.0.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "V2.3.0"
+date:  2020-03-17T10:28:41+01:00
+weight: 1
+---
+# V2.3.0
+
+Sakuli V2.3.0 comes with a lot of improvements in performance and stability as well as with a lot of bug fixes and a
+completely new forwarder integration. Let's have a look at the major changes for this release.
+
+## Sakuli Core
+Beside Node 12 support and a bunch of changes Sakuli click actions, we've improved the CLI output
+in case of check/test failure and added options to ensure a stable website before performing any action.
+A detailed list of all applied enhancement and fixes can be found in the
+[CHANGELOG.md](https://github.com/sakuli/sakuli/blob/develop/CHANGELOG.md).
+
+### Node 12
+From now on, Sakuli supports Node 12. This requires the client systems to update to Node 12 as well, to be
+compatible with the current release. If you use the enterprise container, we already did that for you. The only thing
+you'd have to do is to update to the 2.3.0 container.
+
+### Changes on `_click()` functionality
+A lot of effort has been invested to stabilize the click functionality on websites. In modern web, often
+custom implementation are in place which make it hard for Sakuli to perform click actions on buttons, links, etc.
+Until now, we advised to switch to the native desktop interaction to perform clicks without using the webdriver interface.
+We investigated some of these cases and applied changes to solve these issues. Now Sakulis click action performs
+several technical validations e.g. whether the element to click is visible, before the actual click is performed. If one of
+these checks fails, an error will be thrown. When testing websites using certain technology, the strict checks
+might cause errors, even if the element to click is visible to the end user because it fails technical validation. For those
+cases, it is now possible to avoid all technical checks and validation by passing the configuration object `{force: true}`
+to the click action.
```

Comment:
```suggestion
A lot of effort went into stabilizing the click functionality on websites. 
Clicking a web element might sound trivial at first, but being as heterogenous as it is, the web allows for quite a few edge cases.
Depending on the implementation it might be hard for Sakuli to reliably perform click actions on buttons, links, etc.
Until now, we advised to switch to native desktop interactions in such cases to perform clicks without using the webdriver interface.
For this release, we spent some time on investigating the root cause for these issues. Now Sakuli click actions perform
several technical validations e.g. whether the element to click is visible, before the actual click is performed. If one of
these checks fails, an error will be thrown. On some websites, these strict checks might fail the technical validation.
For those cases, it is still possible to avoid all technical checks and validation by passing the configuration object `{force: true}` to the click action.
```